### ISSUE TITLE
PageTable columns per selected template

### ISSUE THREAD

**Issue by yckart** [ryancramerdesign/ProcessWire#1742]
Would nice to be able to define the shown columns for each selected template.

**Comment by ryancramerdesign** [comment]
I don't think this is feasible because the column headings wouldn't make sense, so would probably have to display separate tables, and at that point the items are no longer sortable among each other. However,  something close to your fallback field definition (rather than per-template definition) might be a good way to accommodate some of it, without adding too much complexity. The column labels would still be out of context when a fallback value was present, but maybe that's okay when it's always thought of as a fallback rather than an isolated table definition.
